package/MDAnalysis/core/groups.py

```diff
@@ -1513,6 +1513,9 @@ def __getitem__(self, item):
         #
         # u.atoms['HT1'] access, otherwise default
         if isinstance(item, string_types):
+            warnings.warn("Using the [] operator with strings is deprecated."
```

Comment:
@kain88-de thanks for adding it back – I must have removed it accidentally when I rebased.